fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/drawer/Drawer.kt

```diff
@@ -849,6 +849,7 @@ fun Drawer(
                     else -> RoundedCornerShape(tokens.borderRadius(drawerInfo))
                 }
             val drawerElevation: Dp = tokens.elevation(drawerInfo)
+            val horizontalDrawerElevation: Dp = 0.dp
```

Comment:
Add a small comment explaining the reason for 0 elevation

Reply:
Set the default value for elevation in token to 0.dp. Ask designer to update the same in Figma spec sheet.

Reply:
Remove the horizontalDrawerElevation, the issue also exists in BottomDrawer as well.

Reply:
Hi, due to version upgrade, I have created a new PR for this issue: https://github.com/microsoft/fluentui-android/pull/501
I have tried to incorporate all your suggestions.